ArrayIndexOutOfBoundsException

When i use /pay player 0.001
![image](https://user-images.githubusercontent.com/26001944/135745443-34eb85cc-58db-4bed-8ac1-507a3936e79a.png)
[16:06:28] [Server thread/INFO]: player1 issued server command: /pay player 0.001
[16:06:28] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'pay' in plugin Economy v2.3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:776) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:1931) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1770) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1751) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(SourceFile:151) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(SourceFile:125) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1148) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1141) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.util.thread.IAsyncTaskHandler.executeAll(SourceFile:110) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1124) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1054) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:305) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
	at me.scruffyboy13.Economy.commands.PayCommand.onCommand(PayCommand.java:66) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.17.1.jar:3241-Spigot-6c1c1b2-1492826]
	... 19 more

--- a/src/me/scruffyboy13/Economy/commands/PayCommand.java
+++ b/src/me/scruffyboy13/Economy/commands/PayCommand.java
@@ -63,7 +63,7 @@ public class PayCommand implements org.bukkit.command.CommandExecutor {
 			}
 			if (amount <= 0) {
 				StringUtils.sendConfigMessage(sender, "messages.money.give.invalidAmount", ImmutableMap.of(
-						"%amount%", args[2]));
+						"%amount%", args[1]));
 				return true;
 			}
 			

